plugins/scaffolder/src/components/ScaffolderPage/ScaffolderPageContextMenu.tsx

```diff
@@ -36,6 +37,7 @@ const useStyles = makeStyles({
 });
 
 export type ScaffolderPageContextMenuProps = {
+  registerNew?: boolean;
```

Comment:
Quick high-level idea, what do you think about `boolean | 'context-menu'` here? :grin:

Reply:
not sure I follow,  can you add it as a suggestion?